src/main/java/frc/robot/Constants.java

```diff
@@ -61,4 +65,23 @@
         public static final double MAX_TELEOP_ACCELERATION = 5; // Maters/second squared
         public static final double MAX_TELEOP_DECELERATION = 9;
     }
+
+    /**
+     * Constants for the Intake
+     */
+    public static class Intake {
```

Comment:
This is fine for now, but if we're building two robots, we're going to want to structure our constants a little bit differently so it's easy to switch between them based on the robot. I can show you how to do this later.

Reply:
Yeah, I'd appreciate help with that